src/data/standards.json

```diff
@@ -1775,6 +1811,21 @@
           }
         ]
       },
+      {
+        "type": "Select",
+        "label": "Phish Spam Action",
+        "name": "standards.SpamFilterPolicy.PhishSpamAction",
+        "values": [
+          {
+            "label": "Quarantine the message",
+            "value": "Quarantine"
+          },
+          {
+            "label": "Move message to Junk Email folder",
+            "value": "MoveToJmf"
+          }
```

Comment:
Could you flip these around so their location is consistent?

Reply:
Hello @OfficialEsco, Thank you very much for the response. I have made updated standards.json based on your feedback.